russian/window/win-exit.md

```diff
@@ -0,0 +1,9 @@
+Как только вы забираетесь на подоконник и уже готовы спрыгнуть, вы понимаете что окно было несущей конструкцией и дом начинает рушиться!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.